wellregistry/registry/admin/fetch_from_nwis.py

```diff
@@ -43,6 +46,17 @@ def _validate_site(site_data):
             return False, 'Site is missing a well depth'
         return True, 'Valid site'
 
+    @staticmethod
+    def _get_local_aquifer_name(list_aqr_lookups, aqfr_cd, state_cd):
+        try:
+            for item in list_aqr_lookups:
```

Comment:
To echo @mbucknell's comments regarding exception handling. In general, I think it would be better to put `try/except` logic in a `for` loop, rather than a loop within a `try`. However, all the dictionaries in `list_aqr_lookups` should have the keys you expect.